### ISSUE TITLE
PLG_FIELDS_SQL_PARAMS_QUERY_DESC language file

### ISSUE THREAD

**Issue by MartijnMaandag** [joomla/joomla-cms#15531]
### Steps to reproduce the issue
In PLG_FIELDS_SQL_PARAMS_QUERY_DESC in xx-XX.plg_fields_sql.ini we have the following text:
"The SQL query which will provide the data for the dropdown list. The query must return two columns; one called 'value' which will hold the values of the list items; the other called 'text' containing the text in the dropdown list."
'value' and 'text' are values that should not be translated.
Can somebody make it variables which can't be translated?
If somebody translates this and somebody uses that Joomla crashes.

### System information (as much as possible)
Joomla 3.7.0 rc4

**Comment by brianteeman** [comment]
Wouldnt it be better to just add a note not to translate those words as we have done here 

https://github.com/joomla/joomla-cms/blob/staging/installation/language/en-GB/en-GB.ini#L122

Thats an easy PR to make

**Comment by MartijnMaandag** [comment]
People using Crowdin (like me) will not see this.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15531">issues.joomla.org/tracker/joomla-cms/15531</a>.</sub>

**Comment by brianteeman** [comment]
@bakual I thought you had resolved that

**Comment by Bakual** [comment]
Adding context is simple in Crowdin. I can do that later.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15531">issues.joomla.org/tracker/joomla-cms/15531</a>.</sub>

**Comment by MartijnMaandag** [comment]
We did it already ;-)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15531">issues.joomla.org/tracker/joomla-cms/15531</a>.</sub>

**Comment by brianteeman** [comment]
So can this be closed or do you want me to add that comment to the language file?

**Comment by MartijnMaandag** [comment]
The best is to add variables to the string. Don't give something to translate if you don't have to translate it.
When you are translating you don't always look at the context in Crowdin (99% it shows the string-identifier). 
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15531">issues.joomla.org/tracker/joomla-cms/15531</a>.</sub>

**Comment by brianteeman** [comment]
ok well i will have to leave that to others then. Seems crazy to me that someone will try and translate something without the context because without the context its never going to give a quality translation

**Comment by MartijnMaandag** [comment]
That's true, but I could not find a translation in Crowdin where it wasnot translated.
I did it to as a start.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15531">issues.joomla.org/tracker/joomla-cms/15531</a>.</sub>

**Comment by Bakual** [comment]
> The best is to add variables to the string.

I disagree with that. Passing a static text with a variable doesn't make much sense. I think it's also simpler for the translators in the end because they know what they deal with. Of course documentation is key and afaik it was lacking.

**Comment by infograf768** [comment]
I think that Brian is right and that we should add a comment before the string as not all TTs anyway use crowdin.
For example, when using com_localise, the comment will display:

![screen shot 2017-04-26 at 08 31 13](https://cloud.githubusercontent.com/assets/869724/25420946/d9068fa0-2a5a-11e7-8246-09ed3f35762b.png)

when the commented line is placed like this (without any blank line before):
```
PLG_FIELDS_SQL_PARAMS_MULTIPLE_LABEL="Multiple"
; In the string below the terms 'value' and 'text' should not be translated
PLG_FIELDS_SQL_PARAMS_QUERY_DESC="The SQL query which will provide the data for the dropdown list. The query must return two columns; one called 'value' which will hold the values of the list items; the other called 'text' containing the text in the dropdown list."
```

**Comment by brianteeman** [comment]
I will do that PR shortly although to be honest if someone is even thinking of writing SQL then they should realise what they need to write and if they write a bad query then it will fail. There is nothing we can currently do to validate the query and there are more ways to write  bad query than just using the translated strng ;)

**Comment by brianteeman** [comment]
Closed see #15571
